docker/ycsb/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:8 as builder
+
+RUN apt-get update && apt-get install -y \
+    git \
+    maven
+
+RUN git clone https://github.com/fruch/YCSB.git -b add_status_count
```

Comment:
address comment to the PR https://github.com/brianfrankcooper/YCSB/pull/1399

Reply:
fix the comment on that PR, waiting for results